lib/ex_unit/lib/ex_unit/assertions.ex

```diff
@@ -392,8 +392,11 @@ defmodule ExUnit.Assertions do
       Regex.regex?(message) -> Exception.message(error) =~ message
     end
 
-    msg = "Wrong message for #{inspect exception}. " <>
-          "Expected #{inspect message}, got #{inspect Exception.message(error)}"
+    msg = "Wrong message for #{inspect exception}\n" <>
```

Comment:
@whatyouhide you (and @josevalim) were right. In Mix error messages that start with capital letter, we don't use a trailing period. I have fixed so many of those, that I forgot what the actual rule was.. haha